### PR TITLE
kelly/ico-faq

### PR DESCRIPTION
**Pull request by kellybinary**

### CODE REVIEW

File: src/templates/landing_pages/ico.html.tt [hunk 1]
```diff
@@ -685,10 +705,23 @@
...
+                            <p>[% l("Second, tokens may be converted into Ordinary Shares of the Company (subject to certain conditions, as described in the Information Memorandum), thereby providing the legal link between tokens and Ordinary Shares.") %]</p>
+                            <p>[% l("Third, token holders will be able to freely trade their tokens. In contrast, Ordinary Shares are not listed and therefore cannot be traded on any exchange. In that sense, the tokens will be a superior investment to Ordinary Shares.") %]</p>
+                        </div>
+                    </div>
+                    <div class="gr-12">
+                        <div class="faq">
+                            <h4>[% l("Where can I view the source code of the tokens?") %]</h4>
+                            <p>[% l("Binary.com intends to deploy the Mini Me token. Please view our blog post detailing our reasons for choosing this token.") %]</p>
```
Comment (jy-binary) [inline src/templates/landing_pages/ico.html.tt]: "Mini Me" should be a hyperlink to https://github.com/Giveth/minime (in a new tab)

Comment (jy-binary) [inline src/templates/landing_pages/ico.html.tt]: "blog post" should be a hyperlink to https://tech.binary.com/an-erc20-securities-token/

File: src/templates/landing_pages/ico.html.tt [hunk 2]
```diff
@@ -662,8 +663,26 @@
...
+                        <div class="faq">
+                            <h4>[% l("How high should I bid?") %]</h4>
+                            <p>[% l("The higher you bid, the more your chances of being successful in the auction. However irrespective of how high you bid, at the end of the auction all winning bids will receive the same final price, which is the lowest winning bid.") %]</p>
+                        </div>
+                    </div>
+                    <div class="gr-12">
+                        <div class="faq">
                             <h4>[% l("Who can purchase the tokens?") %]</h4>
-                            <p>[% l("Tokens will only be offered to [_1] clients who have a Real Money account, and who have completed our [_2]account authentication/KYC process[_3].", broker_name, '<a href="https://ico_documents.binary.com/2017_Authentication_Process.pdf" target="_blank">', '</a>') %]</p>
+                            <p>[% l("Tokens will only be offered to [_1] clients who have a Real Money account. Please visit <a href='[_1]' target='_blank'>www.binary.com</a> to open an account.", broker_name, 'www.binary.com') %]</p>
```
Comment (jy-binary) [inline src/templates/landing_pages/ico.html.tt]: link isn't working - do you need to add https:// to the URL?

Comment (kellybinary) [reply src/templates/landing_pages/ico.html.tt]: i've fixed the link. need to `request.url_for('home')`

**Pull request merged**